Source/Core/Core/HW/GCMemcard.h

```diff
@@ -116,7 +116,7 @@ struct Header           //Offset    Size    Description
 	// end Serial in libogc
 	u8 deviceID[2];     //0x0020    2       0 if formated in slot A 1 if formated in slot B
 	u8 SizeMb[2];       //0x0022    2       Size of memcard in Mbits
-	u16 Encoding;       //0x0024    2       Encoding (ASCII or japanese)
+	u16 Encoding;       //0x0024    2       Encoding (ASCII or Japanese)
```

Comment:
> (18:53:56) Stevoisiak: I don't really want to update the content of the comments in this PR, just make minor corrections

I'm fine with not fixing it in this PR (neither I or Stevoisiak can confirm which encodings actually are used), so don't consider this a blocker.